fdbrpc/include/fdbrpc/LoadBalance.actor.h

```diff
@@ -746,8 +746,9 @@ Future<REPLY_TYPE(Request)> loadBalance(
 			// Only the first location is available.
 			ErrorOr<REPLY_TYPE(Request)> result = wait(firstRequestData.response);
 			if (firstRequestData.checkAndProcessResult(atMostOnce)) {
-				//wait(firstRequestData.maybeDoReplicaComparison(
-				    //stream, request, model, firstRequestData.response, alternatives, channel));
+				// Do consistency check, if requested.
+				wait(firstRequestData.maybeDoReplicaComparison(
```

Comment:
Invoking maybeDoReplicaComparison() is going to be a no-op in this case. But we may require "maybeDoReplicaComparison()" to throw an error if the invoker (strictly) requires a comparison with all replicas.